instrumentation/grails-3.0/javaagent/grails-3.0-javaagent.gradle

```diff
@@ -5,7 +5,7 @@ muzzle {
     group = "org.grails"
     module = "grails-web-url-mappings"
     versions = "[3.0,)"
-    skipVersions += ['3.1.15']
+    skipVersions += ['3.1.15', '3.3.6']
```

Comment:
I saw a recent build fail on `3.3.6` too, but I don't see it in the maven central metadata: https://repo1.maven.org/maven2/org/grails/grails-web-url-mappings/maven-metadata.xml, and I can't reproduce it locally..

Reply:
I pushed this `skipVersions` line to #2611 and will merge there to unblock a couple of PRs that look stuck on this too

Reply:
Grails module has a separate repository, `https://repo.grails.org/grails/core`. It does have `3.3.6` but that has a SNAPSHOT transitive dependency which we fail to retrieve.

Reply:
oh I totally missed that, thx (I just needed to read down a few lines 😭)